packages/blockchain-extension/extension/commands/addWalletIdentityCommand.ts

```diff
@@ -121,22 +119,22 @@ export async function addWalletIdentity(walletItem: WalletTreeItem | FabricWalle
                 }
             };
             // Get the json identity file path
-            const jsonIdentityPath: vscode.Uri = await UserInputUtil.browse('Browse for a JSON identity file', [UserInputUtil.BROWSE_LABEL], openDialogOptions, true) as vscode.Uri;
+            const jsonIdentityPath: vscode.Uri = await UserInputUtil.browse('Browse for a JSON identity file', [UserInputUtil.BROWSE_LABEL], openDialogOptions) as vscode.Uri;
             if (!jsonIdentityPath) {
                 return;
             }
             const certProperty: string = 'cert';
             const privateKeyProperty: string = 'private_key';
-            const jsonIdentityContents: string = await fs.readFile(jsonIdentityPath.fsPath, 'utf8');
-            const jsonIdentity: any = JSON.parse(jsonIdentityContents);
+            const jsonIdentityContents: Uint8Array = await vscode.workspace.fs.readFile(jsonIdentityPath);
```

Comment:
Did we want to put a wrapper around this?
E.g.
```
static async ExtensionUtil.readFile(path: string): Promise<string> {
    const fileContent: Uint8Array = await vscode.workspace.fs.readFile(path);
    return fileContent.toString();
}
```

I think we're likely to remember calling ExtensionUtil.readFile() over vscode.workspace.fs.readFile()